smcontroller/smclient.go

```diff
@@ -33,6 +34,7 @@ import (
 	"google.golang.org/protobuf/types/known/timestamppb"
 
 	"github.com/aoscloud/aos_common/api/cloudprotocol"
+	"github.com/aoscloud/aos_communicationmanager/amqphandler"
```

Comment:
split imports aos_common and aos_communicationmanager

Reply:
done